cmd/soroban-cli/src/commands/tx/xdr.rs

```diff
@@ -29,6 +29,7 @@ pub fn tx_envelope_from_input(input: &Option<OsString>) -> Result<TransactionEnv
             &mut Cursor::new(input.clone().into_encoded_bytes())
         }
     } else {
+        eprint!("waiting for transaction input...\n");
```

Comment:
Should we using the rich print instead, to it follows the same pattern?

Reply:
Updated